operator/v1/types_dns.go

```diff
@@ -287,6 +287,17 @@ type ForwardPlugin struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+	// protocolPolicy configures the protocol policy for upstream DNS requests.
+	//
+	// This field allows a cluster admin to optionally configure:
+	// * None configures CoreDNS for to use its default upstream protocol selection.
+	// * ForceTCP configures CoreDNS to use TCP for all upstream DNS requests, even if the client request uses UDP.
```

Comment:
```suggestion
	// The default value is Default
	// * Default configures CoreDNS to use the default upstream protocol selection.
	// * TCP configures CoreDNS to use TCP for all upstream DNS requests, even if the client request uses UDP.
	// *UDP ....
```

Reply:
So you and Miciah are suggesting two different things - I'm trying to reconcile. I think I captured some of what you suggested here. "Default" is now "" per Miciah's comments. See above response about simplifying the enums to the protocols, I don't think we can really do that here. Mind taking another looks after updates from Miciah's comments?